src/doc/rustc/src/SUMMARY.md

```diff
@@ -15,13 +15,15 @@
 - [Platform Support](platform-support.md)
     - [Template for target-specific documentation](platform-support/TEMPLATE.md)
     - [aarch64-apple-ios-sim](platform-support/aarch64-apple-ios-sim.md)
+    - [aarch64-unknown-none-hermitkernel](platform-support/aarch64-unknown-none-hermitkernel.md)
     - [armv7-unknown-linux-uclibceabi](platform-support/armv7-unknown-linux-uclibceabi.md)
     - [armv7-unknown-linux-uclibceabihf](platform-support/armv7-unknown-linux-uclibceabihf.md)
-    - [aarch64-unknown-none-hermitkernel](platform-support/aarch64-unknown-none-hermitkernel.md)
     - [\*-kmc-solid_\*](platform-support/kmc-solid.md)
```

Comment:
```suggestion
    - [*-kmc-solid_*](platform-support/kmc-solid.md)
```
It's not rendering them correctly with `\` before the star.

Reply:
Tested it locally on mdBook 0.4.15 it doesn't seem to solve it. I tried a few more combinations and it seems that this works:
```suggestion
    - [\*-kmc-solid_\\*](platform-support/kmc-solid.md)
```
as well as
```suggestion
    - [\\*-kmc-solid_\*](platform-support/kmc-solid.md)
```
but I'm not seeing a working option with equal number of `\` or no `\` at all

Reply:
@Michcioperz can you clarify whether one of the suggestions you have written here should be adopted?

Reply:
This is a bug with mdbook.  I posted a fix at https://github.com/rust-lang/mdBook/pull/1785.  I recommend leaving it as the original `[\*-kmc-solid_\*]`.  I'll publish a new version of mdbook soon and update it here and that should fix the rendering.